File: src/test/java/io/bloco/faker/components/AddressTest.java
package io.bloco.faker.components;

import org.junit.Before;
import org.junit.Test;

import io.bloco.faker.Faker;

import static org.junit.Assert.assertNotNull;

public class AddressTest {

    private Faker faker;

    @Before
    public void setUp() {
        faker = new Faker();
    }

    @Test
    public void testCityPrefix() {
        assertNotNull(faker.address.cityPrefix());
    }

    @Test
    public void citySuffix() {
        assertNotNull(faker.address.citySuffix());
    }

    @Test
    public void country() {
        assertNotNull(faker.address.country());
    }

    @Test
    public void countryCode() {
        assertNotNull(faker.address.countryCode());
    }

    @Test
    public void buildingNumber() {
        assertNotNull(faker.address.buildingNumber());
    }
    @Test
    public void streetSuffix() {
        assertNotNull(faker.address.streetSuffix());
    }

    @Test
    public void secondaryAddress() {
        assertNotNull(faker.address.secondaryAddress());
    }

    @Test
    public void postcode() {
        assertNotNull(faker.address.postcode());
    }

    @Test
    public void state() {
        assertNotNull(faker.address.state());
    }

    @Test
    public void stateAbbr() {
        assertNotNull(faker.address.stateAbbr());
    }

    @Test
    public void timeZone() {
        assertNotNull(faker.address.timeZone());
    }

    @Test
    public void city() {
        assertNotNull(faker.address.city());
    }

    @Test
    public void streetName() {
        assertNotNull(faker.address.streetName());
    }

    @Test
    public void streetAddress() {
        assertNotNull(faker.address.streetAddress());
    }

    @Test
    public void defaultCountry() {
        assertNotNull(faker.address.defaultCountry());
    }
}

File: lib/src/test/kotlin/io/bloco/faker/components/AddressTest.kt
package io.bloco.faker.components

import io.bloco.faker.Faker
import org.junit.Assert
import org.junit.Before
import org.junit.Test

class AddressTest {
    private lateinit var faker: Faker
    @Before
    fun setUp() {
        faker = Faker()
    }

    @Test
    fun testCityPrefix() {
        Assert.assertNotNull(faker.address.cityPrefix())
    }

    @Test
    fun citySuffix() {
        Assert.assertNotNull(faker.address.citySuffix())
    }

    @Test
    fun country() {
        Assert.assertNotNull(faker.address.country())
    }

    @Test
    fun countryCode() {
        Assert.assertNotNull(faker.address.countryCode())
    }

    @Test
    fun buildingNumber() {
        Assert.assertNotNull(faker.address.buildingNumber())
    }

    @Test
    fun streetSuffix() {
        Assert.assertNotNull(faker.address.streetSuffix())
    }

    @Test
    fun secondaryAddress() {
        Assert.assertNotNull(faker.address.secondaryAddress())
    }

    @Test
    fun postcode() {
        Assert.assertNotNull(faker.address.postcode())
    }

    @Test
    fun state() {
        Assert.assertNotNull(faker.address.state())
    }

    @Test
    fun stateAbbr() {
        Assert.assertNotNull(faker.address.stateAbbr())
    }

    @Test
    fun timeZone() {
        Assert.assertNotNull(faker.address.timeZone())
    }

    @Test
    fun city() {
        Assert.assertNotNull(faker.address.city())
    }

    @Test
    fun streetName() {
        Assert.assertNotNull(faker.address.streetName())
    }

    @Test
    fun streetAddress() {
        Assert.assertNotNull(faker.address.streetAddress())
    }

    @Test
    fun defaultCountry() {
        Assert.assertNotNull(faker.address.defaultCountry())
    }
}